server/src/test/java/org/elasticsearch/cluster/metadata/MetadataMappingServiceTests.java

```diff
@@ -62,7 +62,7 @@ public void testClusterStateIsNotChangedWithIdenticalMappings() throws Exception
         final MetadataMappingService mappingService = getInstanceFromNode(MetadataMappingService.class);
         final ClusterService clusterService = getInstanceFromNode(ClusterService.class);
         final PutMappingClusterStateUpdateRequest request = new PutMappingClusterStateUpdateRequest("""
-            { "properties" { "field": { "type": "text" }}}""");
+            { "properties": { "field": { "type": "text" }}}""");
```

Comment:
Obviously broken but previously this passed weirdly enough. I'm investigating why separately.

Reply:
https://github.com/elastic/elasticsearch/pull/82408

Reply:
thanks for catching ....